tests/unit/utils/test_uri.py

```diff
@@ -53,5 +57,24 @@ def test_ensure_all_look_like_uri():
     )
 
 
+def test_get_absolute_file_path():
+    """
+    Testing the URI util function `get_absolute_file_path()` returns a normalised
+    path from a string URI which can be used to locate resources on windows as
+    well as unix/linux operating systems.
+    """
+    csv_url = "file:///workspaces/csvcubed/tests/test-cases/cli/inspect/inspector-load-dataframe/pivoted-shape/pivoted-shape-out/testing-converting-a-pivoted-csvw-to-pandas-dataframe.csv"
+    absolute_csv_url = get_absolute_file_path(csv_url)
+    assert isinstance(absolute_csv_url, Path)
+    if os.name == "nt":
```

Comment:
I think you should have different `csv_urls` for each OS. The windows one should probably start with`file:/C:/...`.